certbot/CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 
-*
+* We deprecated support for Apache 2.2 in certbot-apache plugin and it will be
+  removed in the next release of Certbot.
```

Comment:
grammar nit and I think removing Apache 2.2 support requires a major version bump that I wasn't planning on doing quite yet
```suggestion
* We deprecated support for Apache 2.2 in the certbot-apache plugin and it will
  be removed in a future release of Certbot.
```